src/model/base_model.py

```diff
@@ -0,0 +1,82 @@
+import tensorflow as tf
+from tensorflow.keras import layers
+
+
+def fill_fc_weights(fc):
+    if isinstance(fc, tf.keras.Sequential):
+        all_layers = fc.layers
+    else:
+        all_layers = fc
+
+    for m in all_layers:
+        if isinstance(m, layers.Conv2D):
+            if m.use_bias:
+                m.bias_initialier = tf.keras.initializers.constant(0)
```

Comment:
Have you considered passing `bias_initializer` directly to the layer constructor?